esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -889,6 +889,51 @@ void EsdlServiceImpl::handleFinalRequest(IEspContext &context,
                    "No target URL configured for %s!", mthdef.queryMethodName());
     }
 
+    try
+    {
+        Owned<IPTree> responseTree = createPTreeFromXMLString(out, ipt_ordered);
+        auto responseResult = responseTree->queryBranch("//Results/Result");
+
+        if (responseResult)
+        {
+            auto replaceTag = [](StringBuffer& xml, const char* tag, const char* var)
```

Comment:
Is the benefit from using this syntax worth the fact that it might confuse some people?  That is, rather than just creating a normal function?